tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature

```diff
@@ -1,4 +1,5 @@
-@issue-ocis-1317
+@issue-ocis-1317 @skipOnOCIS
+#Share folder has been implemented as Share space in ocis
```

Comment:
```suggestion
# this scenario is skipped on ocis because it makes request to shared folder in root (All Files) which in not possible in OCIS
  # but it works for OC10 see issue https://github.com/owncloud/web/issues/6896 for more detail
```
nothing wrong with the comment you've put but rest of the files have this comment that's why

Reply:
does it matter?

Reply:
I don't think so I'm just suggesting the changes it's eventually up to you to decide if you want to take it or leave it.

Reply:
But I would suggest you at least put the link of the issue or issue tag. It's up to you though